_docs/geyser/api.md

```diff
@@ -81,6 +81,10 @@ like currently, showing emotes.
 The item package contains classes and interfaces related to items in Geyser. You can create custom items and register them using the Geyser API.
 See [here](/geyser/custom-items) for detailed information on how to register custom items.
 
+#### [Block](https://github.com/GeyserMC/Geyser/tree/master/api/src/main/java/org/geysermc/geyser/api/block)
+The item package contains classes and interfaces related to blocks in Geyser. You can create custom blocks and register them using the Geyser API.
```

Comment:
It should be "the block package", not item.
Also, could you change the text so it's different from the item one?
Plus mentioning the custom skulls wouldn't be a bad idea, since those too can be registered in api

Reply:
> It should be "the block package", not item. Also, could you change the text so it's different from the item one? Plus mentioning the custom skulls wouldn't be a bad idea, since those too can be registered in api

The custom skulls is no separate package implementation.What format should I write in better?